templates/datalab/infrastructure-api-deployment.template.yml

```diff
@@ -47,6 +49,15 @@ spec:
               port: 8000
             initialDelaySeconds: 5
             periodSeconds: 10
+        - name: infrastructure-watcher
+          image: {{ &infrastructureApi.imageName }}:{{ datalabVersion }}
+          imagePullPolicy: IfNotPresent
+          command: ["node", "watcher.js"]
+          env:
+            - name: KUBERNETES_MASTER_URL
```

Comment:
Does this actually get used? Given the kubectl proxy shouldn't we be connecting on localhost? I can see that I did the same for the main infrastructure service, but I'm not sure why....

Reply:
Good point, I shall check that.